UAT/V1.10.9_patch.sql

```diff
@@ -0,0 +1,6 @@
+-- https://trello.com/c/AscBN35F/47-18-bulk-upload-local-identifiers-establishment
+
+ALTER TABLE cqc."Establishment" ADD COLUMN "LocalIdentifier" TEXT NULL;
+
+ALTER TABLE ONLY cqc."Establishment"
+    ADD CONSTRAINT "establishment_LocalIdentifier_unq" UNIQUE ("LocalIdentifier");
```

Comment:
This might need a little more clarification with Maria. As it stands, this is a unique constraint across all Establishments. So "Sleazy Pines" may get a duplicate constraint failure for a LOCALID used by "Sunset Care Home", and they'll have no reason for understanding why there is a duplicate.

I think this is going to need to be a partial unique constraint working on `ParentID`

Reply:
Asked Maria, she asked Jackie who confirmed that:
LOCALESTID is workplace so needs to be unique (across the application)
UNIQUEWORKERID is staff level and is unique only across the same workplace.

So I think this is okay to deploy